_templates/component/new/sass-styles.ejs.t

```diff
@@ -0,0 +1,3 @@
+---
+to: src/components/<%= h.inflection.dasherize(component_name) %>/_index.scss
+---
```

Comment:
We should create boiler plate CSS here ```.c-{name} { }```

This file will also change based on the Sass library ticket as it should include the mixin to generate the styles too

Reply:
oh ok, I wasn't sure of what syntax we wanted here based on the ticket

Reply:
updated